ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@ const Yaml = (props: IProps) => {
       setStreaming(false)
```

Comment:
Consider adding error handling or null checks for the contentRef and interpretEndRef references to prevent potential runtime errors if the refs are null or undefined.